### ISSUE TITLE
[BUG] Getting some socket options on a group fails (SRTO_PBKEYLEN, SRTO_KMSTATE, SRTO_CRYPTOMODE)

### ISSUE THREAD

**Issue by maxsharabayko** [Haivision/srt#2918]
Getting some socket option values on a group now fails after PR #2891. 

```
12:54:55.801857 [E] SOCKET::SRT Failed to get sockopt SRTO_PBKEYLEN.
12:54:55.803553 [E] SOCKET::SRT Failed to get sockopt SRTO_KMSTATE.
12:54:55.805568 [E] SOCKET::SRT Failed to get sockopt SRTO_CRYPTOMODE.
```

### Reproduction

srt-xtransmit retrieves those socket option values after a connection is established to form this info log line

```
12:54:55.807114 [D] SOCKET::SRT_GROUP @2020430165 SYNC Group member connected to remote.
    TSBPD off. KM state INVALID (RCV UNSECURED, SND UNSECURED). PB key length: -1. Cryptomode INVALID. Stream ID: not set.
```

### Reasons

`SRTO_CRYPTOMODE` is not set by the caller, so it is not in the `m_config`
```c++
    vector<ConfigItem>::const_iterator i = find_if(m_config.begin(), m_config.end(),
            FOptionValue(optname));

    if (i == m_config.end())
    {
        // Not found, see the defaults
        if (!getOptDefault(optname, (pw_optval), (w_optlen)))
            throw CUDTException(MJ_NOTSUP, MN_INVAL, 0);

        return;
    }
```

**Comment by ethouris** [comment]
Duplicates #2918

**Comment by maxsharabayko** [comment]
Fixed by #2912.
